frontend/src/components/modals/InviteModal.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import { Modal, MessageBoxFunctionalLayout } from 'wix-style-react';
+import { useSelector, useDispatch } from 'react-redux';
+import doAction from '../../state/actions/action';
+import InviteForm from '../auth/InviteForm';
+import Notification from '../Notification';
+import {
+  ACTION_FAILED, ACTION_SUCCEEDED,
+} from '../../constants/ActionStatus';
+
+const InviteModal = ({ isModalOpened, onCloseModal }) => {
+  const [showNotification, setShowNotification] = useState(false);
+
+  const dispatch = useDispatch();
+  const action = useSelector(state => state.action);
+
+  const inviteUser = user => {
+    dispatch(doAction('invites/invite', user));
+  };
+
+  if (action.status === ACTION_SUCCEEDED)
+    setShowNotification(true);
+
+  return (
+    <Modal
+      isOpen={isModalOpened}
+      onRequestClose={() => onCloseModal()}
+      onClose={() => onCloseModal()}
+      shouldDisplayCloseButton
+    >
+      <MessageBoxFunctionalLayout
+        title="Invite new employee"
+        maxHeight="800px"
```

Comment:
I think we need to think about these heights - where to put them, what values to use, because we use these magic pixels in multiple places